src/components/nav-drawer/styles/nav-drawer.base.scss

```
@@ -28,6 +26,36 @@
     [part='mini'] {
         width: rem(60px);
     }
+
+    [part='overlay'] {
+        opacity: .54;
+        background: color(gray, 500);
```

You can actually pass a third parameter to the color function - the alpha. So instead of using opacity for the entire container, you could just do:

```suggestion
        background: color(gray, 500, .54);
```